src/circle2d.jl

```diff
@@ -13,12 +13,15 @@ CircleThreePoints(p1::CartesianIndex{2}, p2::CartesianIndex{2}, p3::CartesianInd
     CircleThreePoints(Point(p1), Point(p2), Point(p3))
 
 function draw!{T<:Colorant}(img::AbstractArray{T, 2}, circle::CircleThreePoints, color::T)
+    ind = indices(img)
     x1 = circle.p1.x; y1 = circle.p1.y
     x2 = circle.p2.x; y2 = circle.p2.y
     x3 = circle.p3.x; y3 = circle.p3.y
     D = det([[x1, x2, x3] [y1, y2, y3] [1, 1, 1]])
-    D != 0 || error("Points do not lie on unique circle")
+    ! isapprox(D, 0, rtol = 0, atol = eps(Float64)) || error("Points do not lie on unique circle")
```

Comment:
Threshold comparison can be surprisingly tricky. Here one issue is that this criterion is not "scale invariant," meaning that I can easily fool the criterion you specified just by multiplying `x1,  x2, x3` by 10. How about making the threshold `maximum(abs, (x1, x2, x3)) * maximum(abs, (y1, y2, y3)) * eps(Float64)`?